uli-website/src/pages/blog/chisel-online-abuse.mdx

```diff
@@ -0,0 +1,33 @@
+---
+name: Designing a Tool to Chisel Away at Online Abuse
+excerpt: "Is it possible to imbue the creation of a #machinelearning tool with principles? That was our intent with Uli."
+author: "Tattle"
+project: 
+date: 02-05-2024
+tags: 
+---
+
+import ContentPageShell from "../../components/molecules/ContentPageShell.jsx"
+
+<ContentPageShell>
+
+![](/HeroIllustration.gif)
+
+Tackling online gender-based violence is a large issue – with policy and legal, behavioural change, tackling patriarchal societal norms, and political will – as just some of the components that need to be addressed.  
+Our question was how could we help address the problem from a machine learning lens; and what did it mean to do that in a participatory manner? 
+Uli, or chisel in Tamil, became our collective effort to help chisel away at this problem. Early input into possible solutions emphasised the importance of a simple and lightweight tool, easy to use and adaptable. 
+On the front end, the result was the Uli plug-in which allows people to track and archive problematic content (useful also for analysing patterns and reporting to law enforcement), as well as redact slurs. It works primarily on X (formerly Twitter) and on several pages of the Chrome and Brave browsers. 
```

Comment:
"It works primarily on X (formerly Twitter) and on several pages of the Chrome and Brave browsers." needs revision.

Maybe "The first iteration worked only on X(formerly twitter) but now works on most webpages on the internet."